src/actions/authActions.jsx

```diff
@@ -109,42 +123,23 @@ export const facebookAuth = () => {
 
 export const googleAuth = () => {
   return (dispatch, getState, { getFirebase, getFirestore }) => {
-    // make async call to database
-
     const firebase = getFirebase();
     const firestore = getFirestore();
-
     const googleAuthProvider = new firebase.auth.GoogleAuthProvider();
-
     firebase
       .auth()
       .signInWithPopup(googleAuthProvider)
       .then((resp) => {
-        const firstName = resp.user.displayName.substr(
-          0,
-          resp.user.displayName.indexOf(" ")
-        );
-        const lastName = resp.user.displayName.substr(
-          resp.user.displayName.indexOf(" ") + 1
+        return addToFirebaseDB(
+          firestore,
+          resp.user.uid,
+          splitFullName(resp.user.displayName, "first"),
```

Comment:
i would just advise you change the db to store first and last name and optionally initials you can parse from the function if you'd like...

but in general, you shouldn't have to store the initials, because the initials are simply a function of the name. that is to say
1. just store the name
2. when you need the initials in the client, just recompute it

Reply:
another reason why storing is bad is because, suppose someone updates their name, then you have to remember to update the initial (even though the way to update initials from a name never changes)

Reply:
thanks a lot, that would be better